agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -157,7 +157,7 @@ function gather_remote_sysinfo_data {
 	local remote_label=$2
 	debug_log "[$script_name]running sysinfo-dump on $remote_host"
 	cmd="ssh $ssh_opts -n $remote_host pbench-remote-sysinfo-dump $sysinfo_path "$sysinfo" $remote_label"
-	debug_log "[$script_name] $cmd > $sysinfo_path/$remote_label$remote_host.tar.xz 2> $sysinfo_path/$remote_label$remote_host.err"
+	debug_log "[$script_name] $cmd > $sysinfo_path/$remote_label$remote_host.tar 2> $sysinfo_path/$remote_label$remote_host.err"
```

Comment:
Looks like you have your work all on one branch.  Let's see if we can tease these out into separate branches and resubmit.